tests-legacy/Unit/Classes/Module/ModuleCoreTest.php

```diff
@@ -91,36 +90,11 @@ public function testDisplayErrorShouldReturnMultipleErrors()
     }
 
     /**
-     * @param int $multistoreCompatibility
-     * @param bool $throwsException
-     *
      * @dataProvider setMultistoreCompatibilityProvider
      */
-    public function testSetAndGetMultistoreCompatibility(int $multistoreCompatibility, bool $throwsException): void
+    public function testSetAndGetMultistoreCompatibility(): void
```

Comment:
```suggestion
    public function testGetDefaultMultistoreCompatibility(): void
```

Reply:
ah yes, and I should remove the dataprovider notion too :D